masonite/routes.py

```diff
@@ -7,13 +7,19 @@
 from pydoc import locate
 
 from config import middleware
-from masonite.exceptions import RouteMiddlewareNotFound
+from masonite.exceptions import RouteMiddlewareNotFound, InvalidRouteCompileException
 
 
 class Route:
     """Route class used to handle routing.
     """
 
+    route_compilers = {
+        'int': r'(\d+)',
```

Comment:
This is inconsistent. I would change this to `'integer'` based on your other defaults below.

Reply:
hmm that would be a breaking change then. We can merge this in and change for 2.1